src/repl/scala/tools/nsc/interpreter/ReplGlobal.scala

```diff
@@ -108,5 +108,13 @@ trait ReplGlobal extends Global {
     }
   }
 
+  trait ReplAnalyzer extends typechecker.Analyzer {
+    override def rootImports(unit: global.CompilationUnit) = super.rootImports(unit)
+  }
+
+  override lazy val analyzer = new { val global: ReplGlobal.this.type = ReplGlobal.this } with ReplAnalyzer
+
+  def languageWildcardImports: List[Symbol] = analyzer.rootImports(NoCompilationUnit)
+
```

Comment:
These are interesting gymnastics, but would it be bad to make `Contexts.rootImports` public?

Reply:
hmm. 🤔